include/kernels/copy_color_spinor.cuh

```diff
@@ -16,6 +16,21 @@ namespace quda
 
   using namespace colorspinor;
 
+  /** Helper function for parity computation */
+  inline int computeParity(const ColorSpinorField &f)
+  {
+
+    // Account for odd-even vs. even-odd site orders
+    int ret_val = f.SiteOrder() == QUDA_ODD_EVEN_SITE_ORDER ? 1 : 0;
+
+    // Account for potential parity flip to access single parity subset QDP-JIT fields
+    if (f.FieldOrder() == QUDA_QDPJIT_FIELD_ORDER && f.SiteSubset() == QUDA_PARITY_SITE_SUBSET) {
```

Comment:
Thinking out loud: I guess this parity flip here should really be predicated on the solve being an odd-odd solve, and not an even-even solve.  If Chroma asked for an even-even solve we'd grab the wrong parity.  Is this something we should support?

Reply:
Addressed in 447482